tests/test_executable.py

```diff
@@ -32,6 +32,10 @@ def test_from_executable(
     _ = application_pid  # Fixture only used for running the application.
 
     if exec_container_image is not None:
+        if "centos:6" in exec_container_image.tags and any("pyperf" in flag for flag in profiler_flags):
+            # don't run PyPerf on the centos:6 image, it fails. And in any case PyPerf can't run on centos:6.
```

Comment:
Do we know why it fails in this case?

Reply:
Yeah, noted in the commit msg & pr desc

Reply:
You mean the "no such file" error? I saw that, but why does it happen?

Reply:
I feel like the condition you added lacks some additional explanations

Reply:
Because there's no `mount` in the container. But I don't care because we don't need to support PyPerf on CentOS 6 since it's a very old kernel.

Reply:
Hmm kk

Reply:
I can add the explanation for the `mount` but there's no real reason behind it. Had I installed `mount`, it'd work

Reply:
Explained in the commit message of the merge